lib/awful/rules.lua

```diff
@@ -1,5 +1,8 @@
 ---------------------------------------------------------------------------
---- Apply rules to clients at startup.
+--- Rules for clients.
+--
+-- This module applies @{rules} to clients during startup (via @{client.manage},
```

Comment:
This is just a random comment, not a hard change request. Why `@{}` instead of "`". IMHO, the latter is much more readable and end up the same in the HTML output.